CONTRIBUTING.md

```diff
@@ -39,9 +39,11 @@ Once the Hermit environment is activated you can type the following to start a
 hot-reloading ftl agent:
 
 ```
-$ ftl devel ./examples/echo ./examples/time
+$ ftl dev ./examples/echo ./examples/time
 ```
 
+You can then view the console at `http://localhost:8892`
```

Comment:
We might want to point people at `http://localhost:5173` since that's what we print out in the `ftl serve` command.

```sh
ftl serve

info: Starting FTL with 1 controller(s) and 0 runner(s)
info:controller0: Web console available at: http://localhost:5173
```

Reply:
Yeh that's what confused me when I started playing with ftl... I'd build/deploy modules but `5173` would show as empty whereas `8892` would show actual deployments. Is `5173` supposed to be showing the actual state for me?

Reply:
I never use 5173 and always use 8892 🤷

Reply:
I actually think we should be showing 8892 not 5173, as 8892 is the listener used in production.

Reply:
Looks like this isn't as simple as changing `consoleURL` in `/frontend/local.go:17` as the console becomes unreachable

Reply:
https://github.com/TBD54566975/ftl/pull/1023 <- this might help

Reply:
Nice. I'll remove the text in this PR about where to find the console, because that was only there because the discrepancy